design/design_km.Rmd

```diff
@@ -140,36 +140,29 @@ library(teal)
 library(hermes)
 library(random.cdisc.data)
 library(dplyr)
```

Comment:
as discussed before in other context I believe, no need to update any `design/*` files. These are not supposed to run. They are in the repo, but not part of the R package (e.g. they are not included in the build and thus not on CRAN etc. in the future)

Reply:
I hear you @danielinteractive 
Would we like those files to be reproducible? If yes, I think we should keep updating them with the newest **breaking** changes in teal.data. So that you can run this code in the future.

Reply:
Thanks @m7pr , no, these files don't need to be reproducible

Reply:
alrighty, will revert changes in design/ files

Reply:
https://github.com/insightsengineering/teal.modules.hermes/pull/342/commits/71f64d450b923be6338cb2c0e10ebb181288061d